src/importexport/capella/tests/data/test1.capx-ref.mscx

```diff
@@ -45,11 +45,6 @@
         </VBox>
       <Measure>
         <voice>
-          <Clef>
-            <concertClefType>G</concertClefType>
-            <transposingClefType>G</transposingClefType>
-            <isHeader>1</isHeader>
-            </Clef>
```

Comment:
...but here, in the second file, the first clef is dropped.

Reply:
However: these first clefs seem to be superfluous too, if they are the default clef for the instrument, arent't they? I just can't find the condition to check there.

Reply:
Perhaps `Staff::defaultClefType()` is what you need?

Reply:
Yes, but how to get to it?

Reply:
In `readCapVoice`, you can of course do `score->staff(staffIdx)`.

In `Capella::readCapxVoice`, it's a lot less clear. But it looks like actually no changes are necessary in that method: it seems that `Capelle::readVoice` and `Capella::readCapxVoice` only construct the `Capella` object, and `readCapVoice` converts a voice from the Capella object to MuseScore. So, adjusting only the latter might be sufficient. I'm not 100% sure though; I haven't researched the situation extremely thoroughly.

Reply:
Yes, for `readCapVoice()` it'd work, `readCVapxVoice()` is the problem